lib/control/v1/conqueso.js

```diff
@@ -6,6 +6,74 @@ const HTTP_OK = 200;
 const HTTP_METHOD_NOT_ALLOWED = 405;
 
 /**
+ * Format the given data as Java properties
+ *
+ * @param {Object} data
+ * @return {String}
+ */
+function makeJavaProperties(data) {
+  const results = [];
+
+  for (const key in data) {
+    if (data.hasOwnProperty(key)) {
+      results.push(key + '=' + data[key]);
+    }
+  }
+
+  return results.join('\n');
+}
+
+/**
+ * Converts Consul addresess to Conqueso addresses
+ *
+ * Addresses returned by the Consul plugin are formatted as
+ * {
+ *   "consul.service.addresses": ["x.x.x.x", "y.y.y.y"]
+ * }
+ *
+ * Addresses returned by Conqueso are formatted as
+ * {
+ *   "conqueso.service.ips": "x.x.x.x, y.y.y.y"
+ * }
+ *
+ * @param {Object} properties  Flattened properties as returned from Storage#properties
+ * @return {Object}             Properties with Consul address converted to Conqueso ones
+ */
+function transformAddresses(properties) {
+  const results = {};
+
+  for (const key in properties) {
+    if (properties.hasOwnProperty(key)) {
+      if (key.startsWith('consul') && key.endsWith('addresses')) {
```

Comment:
Thanks! That's what I wanted to do originally but couldn't figure out how. I refactored the code so Conqueso formatted addresses are not injected into the returned Java properties list based on the list of Consul services.